### PR TITLE
DS-989 | Use modals for changemaker card content for all breakpoints

### PR DESCRIPTION
**Pull request by yvonnetangsu**
# READY FOR REVIEW

# Summary
- Use modals for changemaker card content for all breakpoints

# Review By (Date)
- Soon

# Criticality
- 6

# Review Tasks

## Setup tasks and/or behavior to test

1. Go to https://deploy-preview-373--giving-campaign.netlify.app/changemakers
2. Click on a few of the changemaker cards (both horizontal and vertical variants), check that they open up a modal
3. Check that you can use escape key or click outside the modal to close it
4. Check that focus trap works
5. Test on the phone if possible

# Associated Issues and/or People
- JIRA ticket(s) - DS-989

### CODE REVIEW

File: components/AnnotatedImage/AnnotatedImage.styles.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-export const root = 'relative self-start';
+export const root = 'relative self-start annotated-image';
```
Comment (yvonnetangsu) [inline components/AnnotatedImage/AnnotatedImage.styles.ts:1]: For if Forrest wants to track clicks for a specific group of components in Heap.

**Comment by sherakama** [comment]
share_6941708397218730939

Looks good on Mobile

**Review by sherakama — APPROVED** [review]
GTG

**Comment by yvonnetangsu** [comment]
Thanks @sherakama ! 😄

**Pull request merged**